src/components/Main/MainView/MessageElement/MessageElement.vue

```diff
@@ -39,7 +39,7 @@
       <div :class="$style.stampWrapper">
         <icon
           name="rounded-triangle"
-          :size="24"
+          :size="16"
```

Comment:
```suggestion
          :size="20"
```
24が大きいのはわかるけど16は小さすぎない？

Reply:
良い気がする